### PR TITLE
RocksDB 3.10.2

### PR DESCRIPTION
**Pull request by marpaia**
Upgrading the version of RocksDB to 3.10.2

### CODE REVIEW

File: Library/Formula/rocksdb.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 class Rocksdb < Formula
   homepage "http://rocksdb.org"
-  url "https://github.com/facebook/rocksdb/archive/rocksdb-3.6.1.tar.gz"
-  sha1 "5913cfe18a16487d6b3957fe104354c0ec15b9c7"
+  url "https://github.com/facebook/rocksdb/archive/rocksdb-3.10.2.tar.gz"
+  sha1 "d7515f81d5b9e3f74b340f610f331d1cc990578f"
```
Comment (tdsmith) [inline Library/Formula/rocksdb.rb]: The bot sees a different checksum. Can you use sha256 when you update this?

Comment (marpaia) [inline Library/Formula/rocksdb.rb]: Bah, that's the commit hash, not the sha of the tarball. I'll update.

File: Library/Formula/rocksdb.rb [hunk 2]
```diff
@@ -2,8 +2,8 @@
 
 class Rocksdb < Formula
   homepage "http://rocksdb.org"
-  url "https://github.com/facebook/rocksdb/archive/rocksdb-3.6.1.tar.gz"
-  sha1 "5913cfe18a16487d6b3957fe104354c0ec15b9c7"
+  url "https://github.com/facebook/rocksdb/archive/rocksdb-3.10.2.tar.gz"
+  sha256 "5ace408b12e5e5c836c9ba0b1bd57662784d15820cd02b203459b3ac2e01fde7"
   revision 1
```
Comment (jacknagel) [inline Library/Formula/rocksdb.rb]: remove the revision on version updates (thanks!)

**Comment by tdsmith** [comment]
Thanks!

**Comment by marpaia** [comment]
Thanks, @tdsmith!

**Pull request closed without merging**